fastapi_sqla/_pytest_plugin.py

```diff
@@ -17,16 +17,33 @@
 
 
 @fixture(scope="session")
-def db_url():
-    """Default db url used by depending fixtures.
+def db_host():
+    """Default db host used by depending fixtures.
+
+    When CI key is set in environment variables, it uses `postgres` as host name else, host used is `localhost`
+    """
+
+    return "postgres" if "CI" in os.environ else "localhost"
```

Comment:
Why not check if a `PYTEST_DB_HOST` variable is set and set `postgres` in the CI? Some CI configurations may not be able to offer a `postgres` host so this seems more flexible and less convoluted.

Reply:
This logic was already there. I only moved it and fixing this is out of scope of this PR.

Personally even if we check of PTTEST_DB_HOST we still have to check of local vs ci because locally the host is called `localhost` and on CI it is called `postgres` because docker.

Reply:
> out of scope of this PR

Fair

> still have to check of local vs ci 

What I meant is defaulting to localhost but using `PYTEST_DB_HOST` if it exists

Reply:
Right. Good idea